File: src/com/laamella/intellij6502/asm/AssemblyLexerAdapter.java
package com.laamella.intellij6502.asm;

import com.intellij.lexer.FlexAdapter;

import java.io.Reader;

public class AssemblyLexerAdapter extends FlexAdapter {
    public AssemblyLexerAdapter() {
        super(new AssemblyLexer((Reader) null));
    }
}

File: src/main/kotlin/com/laamella/intellij6502/asm/AssemblyLexerAdapter.kt
package com.laamella.intellij6502.asm

import com.intellij.lexer.FlexAdapter
import com.laamella.intellij6502.asm.AssemblyLexer

import java.io.Reader

class AssemblyLexerAdapter : FlexAdapter(AssemblyLexer(null as Reader?))